src/index.css

```diff
@@ -1,5 +1,5 @@
 /* component styles */
-.rdts-hide:not(.match-in-children) {
+.hide:not(.match-in-children) {
```

Comment:
Nice find!

Agreed about CSS Modules. I think it may need a bit more support from the component itself. Let me see if I can find a balance between keeping the readable classes and being able to switch to CSS Modules if needed.